stingray/crossspectrum.py

```diff
@@ -190,17 +188,18 @@ def rebin(self, df, method="mean"):
             The newly binned cross spectrum
         """
 
-        ## rebin cross spectrum to new resolution
+        # rebin cross spectrum to new resolution
         binfreq, bincs, step_size = utils.rebin_data(self.freq[1:],
```

Comment:
Why are we ignoring the first element (freq[1:]) if `_fourier_cross()`  returns `freqs[freqs > 0]` ?